src/memote/checks/Consistency.jl

```diff
@@ -129,4 +129,56 @@ function model_is_consistent(
     termination_status(opt_model) == OPTIMAL
 end
 
+"""
+$(TYPEDSIGNATURES)
+
+Returns a list of all metabolites that aren't part of any reactions.
+"""
+function find_disconnected_metabolites(model::MetabolicModel)
```

Comment:
@stelmo I wanted to ask before but kinda remembered only now -- wouldn't we want a connected components algorithm to find any larger disconnected groups?

Reply:
hmm should be do-able, probably easiest to do with Graphs.jl, but what would be the use of this? finding metabolic choke points? that could be cool... especially if somehow connected to regulatory features...

Reply:
nah, if you have 2 groups of metabolites like:
```
a → b
b → c
c → a
d → e
e → d
```
then neither of these is actually isolated, but I don't think the model is in a happy state either.

Running a primitive DFS paint should be able to detect that this property isn't violated.

Reply:
so this is just a reversible reaction:
```
d → e
e → d
```
and this one is just a classic thermodynamically infeasible cycle:
```
a → b
b → c
c → a
```

Reply:
yeah but we're talking about the connectivity here :D